packages/infolists/resources/views/components/entry-wrapper/index.blade.php

```diff
@@ -123,6 +123,7 @@
                         Alignment::Right => 'text-right',
                         default => $alignment,
                     },
+                    'min-w-0',
```

Comment:
Wouldn't `grid-cols-1` be a slightly cleaner fix in this case?
It's the equivalent of `grid-template-columns: repeat(1, minmax(0, 1fr));`, which fixes the overflow issue at the grid level (as explained in the article you linked to).

Reply:
Tested and achieves the same result.